myscripts.js

```diff
@@ -1,26 +1,74 @@
-const collection = [];
+// SELECTORS
 
-function Book(title,author) {
-    this.title = title;
-    this.author = author;
-    
+let collection = [];
+const addbtn = document.getElementById('add');
+// title = document.getElementById('title');
+const bookContainer = document.getElementById('bookContainer');
+let storageObject;
+
+const booksHtmlTemplate = (obj) => `<div class='bookCont'>
+  <label class='title'>${obj.title}</label><br>
+  <label id='author'>${obj.author}</label><br>
+  <button class='remove' type='button' onclick= value=''>Remove</button>
+  <hr>
+  </div>`;
+
+const createBook = (arr) => {
+  arr.forEach((element) => {
+    const dinContent = document.createDocumentFragment();
+    const newDiv = document.createElement('div');
+    newDiv.innerHTML = booksHtmlTemplate(element);
+    dinContent.appendChild(newDiv);
+    bookContainer.appendChild(dinContent);
+  });
+};
+
+const update = () => {
+  const elements = document.getElementsByClassName('remove');
+  const bookCont = document.querySelectorAll('.bookCont');
+  collection = JSON.parse(localStorage.getItem('books'));
+  Array.from(elements).forEach((btn, index) => {
+    btn.addEventListener('click', () => {
+      if (bookCont[index].parentNode) {
+        bookCont[index].parentNode.removeChild(bookCont[index]);
```

Comment:
- [ ] I tested the app and when I create two or more book and remove them then refresh the page one of the removed book is still present in the `localStorage`. Kindly investigate this issue.

Reply:
Hi @omar-labana , 

Thank you for the feedback. We have fixed this error as you suggested.